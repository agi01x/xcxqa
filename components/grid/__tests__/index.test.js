import React from 'react';
import { render } from 'enzyme';
import { Col, Row } from '..';

describe('Grid', () => {
  it('should render Col', () => {
    const wrapper = render(
      <Col span="2" />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render Row', () => {
    const wrapper = render(
      <Row />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('renders wrapped Col correctly', () => {
    const MyCol = () => <Col span="12" />;
    const wrapper = render(
      <Row gutter={20}>
        <div>
          <Col span="12" />
        </div>
        <MyCol />
      </Row>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
