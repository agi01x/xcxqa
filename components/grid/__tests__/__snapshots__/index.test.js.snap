// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid renders wrapped Col correctly 1`] = `
<div
  class="ant-row"
  style="margin-left:-10px;margin-right:-10px"
>
  <div>
    <div
      class="ant-col-12"
      style="padding-left:10px;padding-right:10px"
    />
  </div>
  <div
    class="ant-col-12"
    style="padding-left:10px;padding-right:10px"
  />
</div>
`;

exports[`Grid should render Col 1`] = `
<div
  class="ant-col-2"
/>
`;

exports[`Grid should render Row 1`] = `
<div
  class="ant-row"
/>
`;
